apps/src/studio/api.js

```diff
@@ -166,6 +166,14 @@ exports.changeScore = function (id, value) {
   Studio.queueCmd(id, 'changeScore', {'value': value});
 };
 
+exports.getScore = function () {
+  return Studio.playerScore;
+};
+
+exports.setScore = function (value) {
+  Studio.setScore(value);
```

Comment:
It would probably be good to get confirmation from someone who understands the inner workings of playlab better than I to confirm that this is the best approach.